src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -568,7 +603,9 @@ def _get_components_data(self, batteries: Set[int]) -> List[InvBatPair]:
         """
         pairs_data: List[InvBatPair] = []
         working_batteries = (
-            self._all_battery_status.get_working_batteries(batteries) or batteries
+            batteries
+            if use_all
+            else self._all_battery_status.get_working_batteries(batteries) or batteries
```

Comment:
We don't have automatic fallback to all batteries anymore,  so this has to be like below I guess?
```suggestion
            batteries
            if use_all
            else self._all_battery_status.get_working_batteries(batteries)

Reply:
Yes,  I think the reason we are adding the `force` option is so that we can stop the automatic fallback.  Fallback to include broken batteries is possible only with the `force` option.

Earlier we were having automatic fallback just for the gap discharge scenario,  in which case actors are aware of the kind of request,  so they can set the force option there.

Reply:
Updated code, test and releases notes, thanks for clarifying this matter!